.pre-commit-config.yaml

```diff
@@ -3,13 +3,16 @@ ci:
   autofix_commit_msg: "style: pre-commit fixes"
 
 repos:
-  - repo: https://github.com/psf/black
-    rev: 23.3.0
-    hooks:
-      - id: black
-
   - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: "v0.0.276"
     hooks:
```

Comment:
It would be better to keep the migration of `black` to `ruff --fix` in a separate PR.